src/PJ_geoc.c

```diff
@@ -0,0 +1,56 @@
+/******************************************************************************
+ * Project:  PROJ.4
+ * Purpose:  Transfornation from geographic to geocentric latitude and back.
```

Comment:
Transfor_m_ation

Reply:
Well - actually *Conversion*, but yes - wrong, and not even "just a speeling missteak"